scripts/Interfaces.s.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.8.19;
+
```

Comment:
I encountered that in the angle-multisig, maybe better to create interfaces in respective repo to not copy paste evrywhere